pic.js

```diff
@@ -0,0 +1,55 @@
+/* const eventType = 'click';
+const myButton = document.querySelector('#randomize');
+const myCallback = function(event) {
+    console.log('The button was clicked!', event);
+  }
+  myButton.addEventListener('click', myCallback);
+///start 
+ //vra newPic = document.createElement('img');
+ function selectImage() {
+    var x = math.floor((Math.random()*6)+1);
+    document.getElementById(".image").innerHTML=<img src="img/random${x}.jpeg" >;
+}; */
+
+
+function display_random_image() 
+{
```

Comment:
we want you to use the API to get a random picture to be generated from the unsplash website

by changing the src of the picture

read the documentations from this website for more information
https://source.unsplash.com/